app/code/core/Mage/Core/Model/Session/Abstract/Varien.php

```diff
@@ -87,7 +87,7 @@ public function start($sessionName = null)
                 // no break
             default:
                 session_save_path($this->getSessionSavePath());
-                session_module_name($moduleName);
+                session_module_name((string)$moduleName);
```

Comment:
The param of `session_module_name()` can be type string or null. `$moduleName` type can be `Varien_Simplexml_Element|false`. I do not know how to test this. Can someone explain?

Also, `session_save_path($this->getSessionSavePath());` is same and should have the same treatment.

And these also:
line 80: `call_user_func($this->getSessionSavePath());`
line 81: `is_writable($this->getSessionSavePath())`

Reply:
Good point. I can't test anything at the moment.

Reply:
@kiatng @luigifab I would remove the change to this file so that we can merge this PR, is it ok?

Reply:
Sure.

Reply:
done, i'll mark this conversation as resolved